### PR TITLE
fix: QRCode JsonException crash

### LINKED ISSUE #381: QRCodeReader Fragment crashes
**Summary:**
On Scanning a invalid QRCode, app crashes by throwing `JsonSyntaxException`

**Expected behavior:**
It should prompt user to scan a valid QRCode

### PR DESCRIPTION
**Pull request by dilpreet2028**
Fixes #381

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -258,6 +258,7 @@
     <string name="select_mode">Select Mode</string>
     <string name="add_beneficiary_option">Add Third Party Beneficiaries to your account.\nEither
         enter manually or scan account QR code</string>
+    <string name="invalid_qr">Please scan QRCode of other user Savings or Loan Account</string>
```
Comment (therajanmaurya) [inline app/src/main/res/values/strings.xml]: @dilpreet96 Why this message, Why not `Invalid QR code`

Comment (dilpreet2028) [reply app/src/main/res/values/strings.xml]: @therajanmaurya this msg will provide user with a clear instruction i.e. he should scan only Savings and Loans Account QRCode's but `Invalid QR Code` does not give a complete information to user

Comment (therajanmaurya) [reply app/src/main/res/values/strings.xml]: So your message should be like this, `You can't make  action into your account, please scan other user Savings and Loans Account QRCode`

Comment (dilpreet2028) [reply app/src/main/res/values/strings.xml]: @therajanmaurya changes made

**Pull request merged**